blog/2024-04-30-introducing-opencost-plugins/index.mdx

```diff
@@ -0,0 +1,155 @@
+---
+slug: introducing-opencost-plugins
+title: "Introducing OpenCost Plugins"
+authors: [kaiwombacher]
+tags: [external costs, IT cost monitoring, Datadog costs, IT costs]
+---
+
+![OpenCost banner](./img/OpenCost_plugins_2.png)
+
+We are excited to announce that the CNCF OpenCost project has significantly expanded its scope and capabilities beyond existing Kubernetes and cloud costs monitoring. Via the new [OpenCost Plugins](https://github.com/opencost/opencost-plugins) framework, teams are now able to use OpenCost to monitor ALL of their IT spending (infrastructure monitoring, data platforms, cloud services or other tools) in one place. We have kicked things off with a Datadog plugin, but this is just the first of many plugins coming to OpenCost.

```

Comment:
what about "teams are now _increasingly_ able to use OpenCost to monitor ALL of their IT spending"

... just trying to avoid promising too much here out of the box